rosetta-lang/src/main/java/com/regnosys/rosetta/scoping/RosettaScopeProvider.xtend

```diff
@@ -145,14 +146,14 @@ class RosettaScopeProvider extends ImportedNamespaceAwareLocalScopeProvider {
 					switch (context) {
 						Operation: {
 							val receiverType = typeProvider.getRTypeOfSymbol(context.assignRoot)
-							return Scopes.scopeFor(receiverType.allFeatures(context))
+							return Scopes.scopeFor(receiverType.RType.allFeatures(context))
 						}
 						Segment: {
 							val prev = context.prev
 							if (prev !== null) {
 								if (prev.attribute.isResolved) {
 									val receiverType = typeProvider.getRTypeOfSymbol(prev.attribute)
-									return Scopes.scopeFor(receiverType.allFeatures(context))
+									return Scopes.scopeFor(receiverType.RType.allFeatures(context))
```

Comment:
This one and previous one will need to change as part of your next story (for setting)